### PR TITLE
Fixes #11030: Add Python support.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Anybody up for reviewing this?

File: lib/runcible/models/python_importer.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module Runcible
+  module Models
+    class PythonImporter < Importer
+      ID = 'python_importer'
+      REPO_TYPE = 'python-repo'
+
+      attr_accessor 'feed', 'max_speed', 'max_downloads', 'upstream_name',
+                    'proxy_port', 'proxy_password', 'proxy_username', 'mask_id',
+                    'packages_names'
+
```
Comment (parthaa) [inline lib/runcible/models/python_importer.rb]: Any ssl attributes supported here in the importer?

Comment (ehelms) [inline lib/runcible/models/python_importer.rb]: Anything the default importer supports, the Python type supports -- I'm trying to find again the default configs in their documentation to check this.

**Comment by parthaa** [comment]
APJ

**Pull request merged**